guides/roles/fixed-wing.md

```diff
@@ -0,0 +1,129 @@
+# Fixed Wing Pilots
+
+This guide will only cover the use of NATO equipment.
+
+<hr/>
+
+# Basics
+
+This guide will not cover the basics of flight, and only covers operations.
+
+## HMD
```

Comment:
[HMD =/= HUD](https://www.sygic.com/blog/2017/what-is-a-head-up-display-a-k-a-hud)